docs/release.md

```diff
@@ -19,3 +19,58 @@ Releases can be found on the github [release page](https://github.com/CSCfi/rems
 
 The integration test environment is updated by the developers.
 CSC updates all supported environments such as https://rems2demo.csc.fi which is for public testing and customer specific instances.
+
+## Step-by-step instructions for creating a release
+
+1. Checkout the most recent master, e.g.,
+
+   git pull; git checkout origin/master
+
+2. Edit CHANGELOG.md:
+
+   - See the name of the previous release, find the street with that name,
+     select a street that is next to that street but is not yet used as
+     a release name.
+
+   - Add a line for the new release, e.g.,
+     ## v2.6 "Kalevalantie" 2018-11-12
```

Comment:
should probably quote this with `` to stop github from rendering it as an actual heading